util/task/ticker_task_test.go

```diff
@@ -0,0 +1,62 @@
+package task_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/prebid/prebid-server/util/task"
+	"github.com/stretchr/testify/assert"
+)
+
+type MockRunner struct {
+	RunCount int
+}
+
+func (mcc *MockRunner) Run() error {
+	mcc.RunCount++
+	return nil
+}
+
+func TestStartWithSingleRun(t *testing.T) {
+	// Setup:
+	runner := &MockRunner{RunCount: 0}
+	interval := 0 * time.Millisecond
+	ticker := task.NewTickerTask(interval, runner)
+
+	// Execute:
+	ticker.Start()
+	time.Sleep(10 * time.Millisecond)
+
+	// Verify:
+	assert.Equal(t, runner.RunCount, 1, "runner should have run one time")
+}
+
+func TestStartWithPeriodicRun(t *testing.T) {
+	// Setup:
+	runner := &MockRunner{RunCount: 0}
+	interval := 10 * time.Millisecond
+	ticker := task.NewTickerTask(interval, runner)
+
+	// Execute:
+	ticker.Start()
+	time.Sleep(25 * time.Millisecond)
```

Comment:
`TestStop` does make a `ticker.Stop()` call. Are we missing that call here?

Reply:
Adding `Stop()` here makes sense as it would lead to more consistent test results. However, I think we can just combine this test with `TestStop` since testing the stop method is dependent on a task running periodically.

Reply:
I vote to leave the tests separate. `TestStartWithPeriodicRun` is validating an expected run count and adding `Stop()` is just for cleanup. The `TestStop` test has additional wait time to ensure the stop call actually worked. Leaving them separate should point us more clearly to a potential issue.

Reply:
👍 I'll leave the two tests separate and add `Stop()`.